apps/api/v2/src/modules/events/input/get-public-event-input.ts

```diff
@@ -0,0 +1,17 @@
+import { Transform } from "class-transformer";
+import { IsBoolean, IsString } from "class-validator";
+
+export class GetPublicEventInput {
+  @IsString()
+  @Transform(({ value }: { value: string }) => value.toLowerCase())
+  username!: string;
+
+  @IsString()
+  eventSlug!: string;
+
+  @IsBoolean()
+  isTeamEvent?: boolean;
+
+  @IsString()
+  org!: string | null;
```

Comment:
If it must be defined, but string value can't be passed, then null must be supplied. I think it would be better dev experience if org IsOptional instead of explicitly requiring to pass null.